compile.sh

```diff
@@ -40,6 +40,9 @@ _self_dir=$(realpath $(dirname "$0"))
 
 _make_args+=" O=$_out ARCH=arm64 -j$(nproc)"
 
+_cross_compile_32="$ANDROID_ROOT/prebuilts/gcc/linux-x86/arm/arm-linux-androideabi-4.9/bin/arm-linux-androideabi-"
+_make_args+=" CROSS_COMPILE_ARM32=$_cross_compile_32"
```

Comment:
As said in the other comment, I'd prefer is this common code was defined before calling `setup_$_compiler.sh`, but just personal taste.

Reply:
Ah, I might have misunderstood that then. You indeed specifically mentioned "without removing the `_make_vars` variable assignment in line 18" though I assumed it was fine to move it around.

It is indeed nicer to keep the arguments ordered like this in addition to always appending in `setup_*.sh` (so that nothing gets lost in the unexpected event that we want to assign more variables to `_make_args` earlier on in the chain).